### ISSUE TITLE
Añadir vídeos a la wiki

### ISSUE THREAD

**Issue by jasvazquez** [aosucas499/minino-TDE#42]
Álvaro García ha subido, a [su canal de Youtube](https://www.youtube.com/playlist?list=PLTInVXbjD9zvZWyeZvtz3zlu6d77V_zKf), una serie de vídeos explicando algunas de las dudas más habituales que tienen los compañer@s.

Destacar el que explica cómo utilizar Etcher para grabar la ISO de Minino.

¿Te parece que incluyamos aquellos que nos resulten de interés a la wiki @aosucas499? 🤔

**Comment by aosucas499** [comment]
Es una idea genial, menudo crack. Añádela a la wiki al lugar que veas mejor.

**Comment by jasvazquez** [comment]
Anoto revisar los vídeos e incluirlos según proceda (temo que alguno haya dejado de funcionar tras cambiar a Refracta)
Me alegra que te haya gustado 👍🏻

**Comment by aosucas499** [comment]
Yo hoy he estado instalando minino en PC's del centro y la instalación con refracta no es lo que más me gusta, he recordado porqué cambié a systemback. Algunas veces supone que el usb es /dev/sda y te instala el sistema en el pendrive. También leí de pasada comentarios hoy de gente en el grupo de telegram de que se le quedaba pillado en la selección de la hora. Total, que para uso personal me voy hacer una versión con refracta que tengo que instalarlo en 30 PC's chuck norris y así no cunde.

**Comment by jasvazquez** [comment]
¿Puedo entender que cambiaremos la ISO al sistema que menos problema dé a los compañeros/as? 🤔

**Comment by aosucas499** [comment]
Sí, a ver qué cuentan. Yo me haré una iso con systemback privada, para mi uso en el centro. A mí no me cuesta utilizar systemback jaja

**Comment by jasvazquez** [comment]
Suelto la tarea por no haberle podido dedicar tiempo hasta ahora y existir otras donde podría ser más "rentable" mi tiempo.
Si nadie se anima a hacerla, la retomaré más adelante

**Comment by jasvazquez** [comment]
Degrado la tarea a "simple" IDEA tras ver que la carga actual de trabajo desaconseja (e impide) abordarla en el corto plazo (amén de que puede que cambie el instalador y queden obsoletos los vídeos que podamos subir)
